`serialize` feature requires `bevy_render`

Not going to lie, this is actually kind of amazing.

```toml
[dependencies]
avian2d = { git = "https://github.com/Jondolf/avian.git", rev = "c4840dd", default-features = false, features = ["2d", "f32", "serialize"] }
bevy = { version = "0.15.0", default-features = false }
```

```rust
error[E0277]: the trait bound `MassProperties2d: collision::_::_serde::Serialize` is not satisfied
   --> /alercah/.cargo/git/checkouts/avian-b0a258754c9e72b0/c4840dd/crates/avian2d/../../src/dynamics/rigid_body/mass_properties/components/collider.rs:77:42
    |
77  | #[cfg_attr(feature = "serialize", derive(serde::Serialize, serde::Deserialize))]
    |                                          ^^^^^^^^^^^^^^^^ the trait `collision::_::_serde::Serialize` is not implemented for `MassProperties2d`
...
80  | pub struct ColliderMassProperties(MassProperties);
    |                                   -------------- required by a bound introduced by this call
    |
    = note: for local types consider adding `#[derive(serde::Serialize)]` to your `MassProperties2d` type
    = note: for types from other crates check whether the crate offers a `serde` feature flag
    = help: the following other types implement trait `collision::_::_serde::Serialize`:
              &'a T
              &'a mut T
              ()
              (T,)
              (T0, T1)
              (T0, T1, T2)
              (T0, T1, T2, T3)
              (T0, T1, T2, T3, T4)
            and 502 others
note: required by a bound in `serialize_newtype_struct`
   --> /.cargo/registry/src/index.crates.io-6f17d22bba15001f/serde-1.0.216/src/ser/mod.rs:907:21
    |
901 |     fn serialize_newtype_struct<T>(
    |        ------------------------ required by a bound in this associated function
...
907 |         T: ?Sized + Serialize;
    |                     ^^^^^^^^^ required by this bound in `Serializer::serialize_newtype_struct`
    = note: this error originates in the derive macro `serde::Serialize` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0277]: the trait bound `MassProperties2d: Deserialize<'_>` is not satisfied
  --> /.cargo/git/checkouts/avian-b0a258754c9e72b0/c4840dd/crates/avian2d/../../src/dynamics/rigid_body/mass_properties/components/collider.rs:80:35
   |
80 | pub struct ColliderMassProperties(MassProperties);
   |                                   ^^^^^^^^^^^^^^ the trait `Deserialize<'_>` is not implemented for `MassProperties2d`
   |
   = note: for local types consider adding `#[derive(serde::Deserialize)]` to your `MassProperties2d` type
   = note: for types from other crates check whether the crate offers a `serde` feature flag
   = help: the following other types implement trait `Deserialize<'de>`:
             &'a [u8]
             &'a std::path::Path
             &'a str
             ()
             (T,)
             (T0, T1)
             (T0, T1, T2)
             (T0, T1, T2, T3)
           and 505 others

error[E0277]: the trait bound `MassProperties2d: Deserialize<'_>` is not satisfied
    --> /.cargo/git/checkouts/avian-b0a258754c9e72b0/c4840dd/crates/avian2d/../../src/dynamics/rigid_body/mass_properties/components/collider.rs:80:35
     |
80   | pub struct ColliderMassProperties(MassProperties);
     |                                   ^^^^^^^^^^^^^^ the trait `Deserialize<'_>` is not implemented for `MassProperties2d`
     |
     = note: for local types consider adding `#[derive(serde::Deserialize)]` to your `MassProperties2d` type
     = note: for types from other crates check whether the crate offers a `serde` feature flag
     = help: the following other types implement trait `Deserialize<'de>`:
               &'a [u8]
               &'a std::path::Path
               &'a str
               ()
               (T,)
               (T0, T1)
               (T0, T1, T2)
               (T0, T1, T2, T3)
             and 505 others
note: required by a bound in `next_element`
    --> /.cargo/registry/src/index.crates.io-6f17d22bba15001f/serde-1.0.216/src/de/mod.rs:1732:12
     |
1730 |     fn next_element<T>(&mut self) -> Result<Option<T>, Self::Error>
     |        ------------ required by a bound in this associated function
1731 |     where
1732 |         T: Deserialize<'de>,
     |            ^^^^^^^^^^^^^^^^ required by this bound in `SeqAccess::next_element`

For more information about this error, try `rustc --explain E0277`.
error: could not compile `avian2d` (lib) due to 3 previous errors
```

--- a/crates/avian2d/Cargo.toml
+++ b/crates/avian2d/Cargo.toml
@@ -47,6 +47,7 @@ bevy_picking = ["bevy/bevy_picking"]
 serialize = [
     "dep:serde",
     "bevy/serialize",
+    "bevy_heavy/serialize",
     "bevy_transform_interpolation/serialize",
     "parry2d?/serde-serialize",
     "parry2d-f64?/serde-serialize",

